packages/atomic/src/components/insight/user-actions/atomic-insight-user-actions-timeline/e2e/page-object.ts

```diff
@@ -0,0 +1,83 @@
+import {Page} from '@playwright/test';
+import {BasePageObject} from '../../../../../../playwright-utils/base-page-object';
+
+export class UserActionsTimelinePageObject extends BasePageObject<'atomic-insight-user-actions-timeline'> {
+  constructor(page: Page) {
+    super(page, 'atomic-insight-user-actions-timeline');
+  }
+
+  get activeSession() {
+    return this.page.locator('[data-testid="active-session"]');
+  }
+
+  get followingSession() {
+    return this.page.locator('[data-testid="following-session"]');
+  }
+
+  get precedingSession() {
+    return this.page.locator('[data-testid="preceding-session"]');
+  }
+
+  get showFollowingSessionsbutton() {
+    return this.page.locator('[data-testid="show-following-sessions"] button');
```

Comment:
Ideally, you should prefer user-facing locators. In this case [getByLabel()](https://playwright.dev/docs/locators#locate-by-label) could be a good fit since you have labels with the same values.

It is ok to use some not user-facing locators in some situations, but using no user-facing locator at all should be avoided.